db/migrate/20250109170132_add_cached_membership_years_to_people.rb

```diff
@@ -0,0 +1,5 @@
+class AddCachedMembershipYearsToPeople < ActiveRecord::Migration[7.0]
+  def change
+    add_column :people, :cached_membership_years, :integer
```

Comment:
sollten wir noch ein `default: 0, null: false` setzen? In den fixtures haben wir auch den 0 Wert

Reply:
macht Sinn :+1: